data/hai/unfettered jobs.txt

```diff
@@ -0,0 +1,458 @@
+	to offer
+		has "Unfettered Jump Drive 2: offered"
+		random < 80
+	cargo "food (tribute)" 25 2 .05
+	on complete
+		payment 5000 1600
+		dialog phrase "unfettered tribute payment dialog"
+	source
+		attributes "unfettered"
+	stopover
+		distance 3 4
+		attributes "hai"
+		attributes "spaceport"
+	on visit
+		dialog phrase "generic cargo on visit"
+	on stopover
+		dialog phrase "unfettered tribute pickup dialog"
+
+mission "Unfettered Tribute 2"
+	name "Hai Tribute to <planet>"
+	job
+	repeat
+	description "Collect a tribute payment of <tons> of food from <stopovers>, and bring it to the Unfettered on <destination>. Your share of the tribute will be <payment>."
+	to offer
+		has "Unfettered Jump Drive 2: offered"
+		random < 50
+	cargo "food (tribute)" 35 2 .05
+	on complete
+		payment 7000 2000
+		dialog phrase "unfettered tribute payment dialog"
+	source
+		attributes "unfettered"
+	stopover
+		distance 4 5
+		attributes "hai"
+		attributes "spaceport"
+	on visit
+		dialog phrase "generic cargo on visit"
+	on stopover
+		dialog phrase "unfettered tribute pickup dialog"
+
+mission "Unfettered Tribute 3"
+	name "Hai Tribute to <planet>"
+	job
+	repeat
+	description "Collect a tribute payment of <tons> of food from <stopovers>, and bring it to the Unfettered on <destination>. Your share of the tribute will be <payment>."
+	to offer
+		has "Unfettered Jump Drive 2: offered"
+		random < 20
+	cargo "food (tribute)" 45 2 .05
+	on complete
+		payment 9000 2400
+		dialog phrase "unfettered tribute payment dialog"
+	source
+		attributes "unfettered"
+	stopover
+		distance 5 6
+		attributes "hai"
+		attributes "spaceport"
+	on visit
+		dialog phrase "generic cargo on visit"
+	on stopover
+		dialog phrase "unfettered tribute pickup dialog"
+
+
+
+mission "Proud Unfettered Boarded"
+	boarding
+	repeat
+	minor
+	to offer
+		"flagship crew" > 120
+		random < "flagship crew" / 8
+	source
+		government "Hai (Unfettered)"
+		category "Heavy Warship"
+	on offer
+		conversation
+			`As you board the <origin> you immediately notice something is wrong: the ship is starting to explode! The captain sends you a message. "This is not a fair fight. Better to die with my crew standing, than risk being slaughtered by inferior humans! You will not be allowed to capture this ship and use it against our brothers and sisters. I will take you down with us!"`
+				flee
+	destination "Darkcloak"
+
+
+
+mission "Unfettered Silicon Asteroid Mining"
+	name `Gather silicon from asteroids`
+	description `Mine and then collect 50 tons of silicon from the asteroid field in Wah Yoot and then bring them back to <destination> for <payment>.`
+	job
+	repeat
+	source "Darkcloak"
+	to offer
+		has "First Contact: Unfettered: declined"
+		random < 60
+	on complete
+		payment 250000
+		dialog `The Unfettered unloading the materials you mined seem enthused. "This will allow us to repair our ships!" they tell you as they hand over your payment of <payment> credits.`
```

Comment:
You're completely right, I wasn't even sure I would add these missions at first because we don't have the option to ask for a random amount of outfit; like a mineral. But yeah it needs some different dialogs for the different amounts and better ones.
Please do be as critical as you can be!